PocketKit/Sources/Localization/Resources/en.lproj/Localizable.strings

```diff
@@ -194,6 +194,16 @@
 "reader.activity.unfavorite" = "Unfavorite";
 "reader.activity.moveToSaves" = "Move to Saves";
 
+// Reader Settings
+"reader.settings.title" = "Display settings";
+"reader.settings.fontLabel"  = "Font";
+"reader.settings.fontSizeLabel" = "Font size";
+"reader.settings.lineHeightLabel" = "Line Height";
```

Comment:
Nitpick: "Font size" vs "Line Height" - we should pick consistent capitalization. Also, thanks for moving this into more consistent keys!

Reply:
Indeed we should! Thanks for catching this @dskuza !